packages/near-wallet-selector/src/wallets/Wallet.ts

```diff
@@ -61,6 +61,7 @@ export interface BrowserWallet extends BaseWallet {
 
 export interface InjectedWallet extends BaseWallet {
   type: "injected";
+  downloadLink: string;
```

Comment:
Really minor but could we call this `downloadUrl` to align with `iconUrl`?

Reply:
Sure, I did not notice it :)